torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2978,6 +3006,14 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                        SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_extremal',
                                 active_if=IS_WINDOWS),
                    )),
+    OpInfo('gradient',
+           dtypes=all_types_and_complex_and(torch.float16, torch.bfloat16),
+           skips=(
+               SkipInfo('TestOpInfo', 'test_supported_dtypes', dtypes=[torch.uint8]),
+           ),
+           supports_out=False,
```

Comment:
This would be a good follow-up task

Reply:
filed an issue : https://github.com/pytorch/pytorch/issues/56131